src/settings/tabenvironment.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * class provides services:
+ *  get access to configuration file;
+ *  changing application style
+ */
+#ifndef TABENVIRONMENT_H
+#define TABENVIRONMENT_H
+
+#include <QWidget>
+#include <QtWidgets>
+
+class TabEnvironment : public QWidget
```

Comment:
Header QtWidgets includes all elements of Qt UI
It is huge
You must remove this include from your files, especially - from headers
**_It might cause identifiers conflict_**

Reply:
removed